WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -49,7 +50,8 @@ class PostListItemViewHolder(
         setTextOrHide(tvTitle, item.data.title)
         setTextOrHide(tvExcerpt, item.data.excerpt)
         setTextOrHide(tvDateAndAuthor, item.data.dateAndAuthor)
-        setTextOrHide(tvStatusLabels, item.data.statusLabels)
+        uiHelpers.updateVisibility(tvStatusLabels, item.data.statusLabels.isNotEmpty())
```

Comment:
Out of scope: I think we might have misnamed `tvStatusLabels`. At a glance, I thought it was an array of labels (`TextView`). Perhaps `tvStatusesLabel` would have been more appropriate. 🤷‍♂️ The `statusLabels` could have been `statuses` too, I guess.

Reply:
Good point. I'll rename it in another PR.

Reply:
Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9514